File: Segundo Semestre/Estrutura de Dados/Ponteiro/ex004.c
//Crie um programa que contenha uma função que permita passar por parâmetro dois
//números inteiros A e B. A função deverá calcular a soma entre estes dois números e
//armazenar o resultado na variável A. Esta função não deverá possuir retorno, mas deverá
//modificar o valor do primeiro parâmetro. Imprima os valores atualizados de A e B na
//função principal.
#include<stdio.h>

int soma(int *a, int b){
    *a = *a + b;

}

int main(){
    int nA, nB;
    printf("Digite um valor para x: ");
    scanf("%i", &nA);
    printf("Digite um valor para y: ");
    scanf("%i", &nB);
    soma(&nA, nB);
    printf("Soma: %i", nA);

}

File: Segundo Semestre/Dev de Sites Dinamicos/Aula03/ex004.js
//Crie uma função que receba um array de números como parâmetro e retorne a soma de
//todos os elementos do array.
var numeros = [10, 10, 10];

function soma(numeros){
    var total = 0;
    for(var i = 0; i < numeros.length; i++){
        total = total + numeros[i];
    }
    return total;

}
console.log(soma(numeros));